contents/blog/posthog-vs-hotjar.md

```diff
@@ -0,0 +1,538 @@
+            <td>In-app widget for requesting customer interviews.</td>
+        </tr>
+        <tr>
+            <td><strong>Custom user targeting</strong></td>
+            <td className="text-center">Requires additional integration</td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td>Define who should see your widgets with your own data.</td>
+        </tr>
+        <tr>
+            <td><strong>Surveys</strong></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td className="text-center"><span className="text-red text-lg">✖</span></td>
+            <td>Capture user feedback with surveys. </td>
+        </tr>
+        <tr>
+            <td><strong>User research tooling</strong></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td className="text-center"><span className="text-red text-lg">✖</span></td>
+            <td>Integrated tools to assist you in research sessions.</td>
+        </tr>
+        <tr>
+            <td><strong>Notification bar</strong></td>
+            <td className="text-center"><span className="text-red text-lg">✖</span></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td>Display a customised banner on your site.</td>
+        </tr>
+        <tr>
+            <td><strong>In-app popups</strong></td>
+            <td className="text-center"><span className="text-red text-lg">✖</span></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td>Highlight features in your app.</td>
+        </tr>
+    </tbody>
+</table>
+</div>
+
+### Apps and integrations
+Both Hotjar and PostHog have a wide selection of apps and integration. Both also include integrations with tools such as Zapier, which enable you to move data to even more platforms.
+
+One unique advantage of PostHog is that, because it is open source, it’s easy to [create your own apps and integrations](/tutorials/build-your-own-posthog-app). This is useful if you’re, for example, using niche software in your stack that hasn’t been widely adopted, or if you require a direct integration between PostHog and your product. 
+
+
+Below, we've listed a few of the most popular integrations used across PostHog and Pendo. 
+
+<div className="overflow-x-auto -mx-5 px-5">
+<table className="w-full mt-4" style="min-width: 600px;">
+    <thead>
+        <tr>
+            <td className="w-3/12"></td>
+            <td><strong>Hotjar</strong></td>
+            <td><strong>PostHog</strong></td>
+            <td></td>
+        </tr>
+    </thead>
+    <tbody>
+        <tr>
+            <td><strong>Hubspot</strong></td>
+            <td className="text-center"><span className="text-red text-lg">✖</span> Surveys only</td>
```

Comment:
I'd just say no here, and maybe move the 'Surveys only' point to the notes column. It's confusing as-is, because we're defining a Hubspot app as a way to sync data, saying HotJar doesn't have one, but it does have one, just one that doesn't fit that definition.